modulefiles/landda_hera.intel.lua

```diff
@@ -76,6 +76,7 @@ load(pathJoin("mapl", mapl_ver))
 
 load("ufs-pyenv")
 load("atlas")
+load("netcdf-hdf5parallel/4.7.4")
```

Comment:
@ulmononian can you review this change to confirm the package within spack stack?

Reply:
this package is not provided as part of the spack-stack. i checked and it looks like a system default. i would not recommend using it.